job_executor/model/datastore.py

```diff
@@ -245,20 +245,26 @@ def remove(self, job_id: str, dataset_name: str, description: str):
             job_service.update_job_status(job_id, "completed")
             self._log(job_id, "completed")
 
-    def delete_draft(self, job_id: str, dataset_name: str):
+    def delete_draft(self, job_id: str, dataset_name: str, rollback_remove: bool):
         """
         Delete a dataset from the draft version of the datastore.
         """
         self._log(job_id, "initiated")
         job_service.update_job_status(job_id, "initiated")
         dataset_is_draft = self.draft_version.contains(dataset_name)
-        dataset_operation = self.draft_version.get_dataset_operation(
-            dataset_name
-        )
-        if not dataset_is_draft:
+        dataset_operation = self.draft_version.get_dataset_operation(dataset_name)
```

Comment:
I wonder if your linter might not be running ? 🤔 Cause this line is above the linter line length

Reply:
You are absolutely right! Updated now 👍🏻